CONTRIBUTING.md

```diff
@@ -230,30 +230,30 @@ In addition, the following style guidelines are recommended:
+| Network                | Real-world Shield                                                                                          | Americana Representation                                  |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
+| NJ Turnpike            | <img src="https://upload.wikimedia.org/wikipedia/commons/a/af/New_Jersey_Turnpike_Shield.svg" width="40"/> | <img src="icons/shield_us_nj_njtp_noref.svg" width="40"/> |
+| Washington State Route | <img src="https://upload.wikimedia.org/wikipedia/commons/5/52/WA-blank.svg" width="40"/>                   | <img src="icons/shield_us_wa.svg" width="40"/>            |
 
 More complicated shields may be more challenging to simplify. Consider taking some inspiration from the [rebusurance](https://github.com/1ec5/rebusurance) project, which effectivly simplifies a number of complex state shield designs by stretching, compressing, omitting, or simplifying graphic elements.
 
 ### Color Scheme
 
 For consistency, shields should use the following color palette:
 
-| color                                                                 | Pantone | RGB         | Hex triplet |
-| --------------------------------------------------------------------- | ------- | ----------- | ----------- |
-| <img src="doc-img/pantone_294.svg" height=18 width=50 /> Blue         | 294     | 0 63 135    | #003f87     |
-| <img src="doc-img/pantone_469.svg" height=18 width=50 /> Brown        | 469     | 105 63 35   | #693f23     |
-| <img src="doc-img/pantone_342.svg" height=18 width=50 /> Green        | 342     | 0 103 71    | #006747     |
-| <img src="doc-img/pantone_152.svg" height=18 width=50 /> Orange       | 152     | 243 143 0   | #f38f00     |
-| <img src="doc-img/pantone_198.svg" height=18 width=50 /> Pink         | 198     | 223 70 97   | #df4661     |
-| <img src="doc-img/pantone_259.svg" height=18 width=50 /> Purple       | 259     | 109 32 119  | #6d2077     |
-| <img src="doc-img/pantone_187.svg" height=18 width=50 /> Red          | 187     | 191 32 51   | #bf2033     |
-| <img src="doc-img/pantone_116.svg" height=18 width=50 /> Yellow       | 116     | 255 205 0   | #ffcd00     |
-| <img src="doc-img/pantone_382.svg" height=18 width=50 /> Yellow-Green | 382     | 196 214 0   | #c4d600     |
-| <img src="doc-img/white.svg" height=18 width=50 /> White              | N/A     | 255 255 255 | #ffffff     |
-| <img src="doc-img/black.svg" height=18 width=50 /> Black              | N/A     | 0 0 0       | #000000     |
+| Color                                                              | Pantone | RGB         | Hex triplet |
+| ------------------------------------------------------------------ | ------- | ----------- | ----------- |
+| ${\color[RGB]{0,   63,  135}\rule[-4px]{50px}{18px}}$ Blue         | 294     | 0 63 135    | #003f87     |
```

Comment:
Oh that’s weird, now GitHub is unhappy with this syntax, but it was fine with it during preview:

<img src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/8532b492-4c57-4fc1-9c4a-12e89b656165" width="245" alt="Unable to render expression.">

<img src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/ea2f5cc3-fc83-4437-89eb-c59591575420" width="168" alt="Blue">

LaTeX is [supposed to be supported](https://github.blog/2022-05-19-math-support-in-markdown/) these days.

Reply:
Possibly related to orgs/community#45276.

Reply:
Amazing to see bit rot with such a short half-life...

Reply:
Wonder if simplifying the expression helps:

${\color[RGB]{0,63,135}\blacksquare}$

Reply:
I'm seeing a black square in Chrome.

Reply:
Yes, but if you’re using Chrome, you aren’t seeing the blue rectangle a few comments above.

I think this is simply a bug in either GitHub’s MathJax wrapper or a quirk of how they’ve configured MathJax. They understandably have a lot of code that tries to sanitize and otherwise restrict math expressions in Markdown. However, it would make no sense for these restrictions to apply to each browser differently (and not at all to Safari).

Reply:
GitHub is sanitizing the `\rule` macro pretty heavily. I was using it to approximate the dimensions of the images that were being replaced. I could still use `\LARGE` (not `\huge`), but I don’t think it’s really worth it.

Reply:
A few minutes later, the `\blacksquare` macros stopped working in Firefox, but they’re still working in Chrome and Safari. Sigh…

Reply:
Turns out these errors are only occurring in _my_ copy of Firefox, because I had previously activated [MathJax’s accessibility mode](https://docs.mathjax.org/en/latest/basic/accessibility.html) somewhere on GitHub. But if I view this page in a private browser, everything is fine.